szar/clustering.py

```diff
@@ -125,6 +126,28 @@ def Norm_Sfunc(self,fsky):
         ans = self.HMF.dVdz*nbar**2*np.diff(self.HMF.zarr_edges)
         return ans*4.*np.pi*fsky
 
+    def fine_sfunc(self, fsky, nsubsamples):
+        zs = self.HMF.zarr
+        zgridedges = self.HMF.zarr_edges
+
+        values = np.empty(zs.size)
+        for i in range(zs.size):
+            if i == 0:
+                fine_zs = np.linspace(zs[i], zgridedges[i+1], nsubsamples)
+            elif i == zs.size - 1:
+                fine_zs = np.linspace(zgridedges[i], zs[i])
+            else:
+                fine_zs = np.linspace(zgridedges[i], zgridedges[i+1], nsubsamples)
+            ntils = self.ntilde_interpol(fine_zs)
+            dvdz = self.dVdz_fine(fine_zs)
+            integral = simps(dvdz * ntils**2, fine_zs)
+            values[i] = integral * 4 * np.pi * fsky
+        return values
+
+    def v0(self, fsky):
```

Comment:
This function should be an integral over dVdz_fine bounded by zbin start to zbin end

Reply:
Weird. I thought when I pushed my commit to my master it would update that. Will fix.

Reply:
Fixed.